frontend/src/react/Instructions/Instructions.js

```diff
@@ -26,18 +31,62 @@ export default class Instructions extends Component {
         this.setState({[e.target.name]:e.target.value});
     };
 
+    /**
+     * Send an exam to the API and redirect to /newexam/:examId/exercices if success
+     * @return undefined
+     */
+    addExamAndRedirect = async () => {
+        const { title,
+                reminder,
+                instruction} = this.state;
+        const examData = {
+            title,
+            reminder,
+            instruction
+        };
+
+        await this.props.createExam(examData);
+        if (this.props.exams.exam._id) {
+            this.setState({
+                idRedirect: this.props.exams.exam._id,
+                redirectExercices: true})
+        }
+    };
+
+    triggerActive = (e) => {
+        e.target.className = e.target.className + " is-active";
+    };
+
+    triggerInactive = (e) => {
+        e.target.className = e.target.className.substr(0, e.target.className.length - 10);
```

Comment:
Je propose plutôt : 
`e.target.className = e.taget.className.replace('is-active', '');`

Reply:
Done :rocket: